package.json

```diff
@@ -90,13 +90,19 @@
     "text-mask-addons": "^3.8.0"
   },
   "peerDependencies": {
+    "@emotion/react": "^11.9.3",
+    "@emotion/styled": "^11.9.3",
     "@material-ui/core": ">= 4 < 5",
     "@material-ui/icons": ">= 4 < 5",
     "@material-ui/lab": "^4.0.0-alpha.46",
     "@material-ui/pickers": "^4.0.0-alpha.12",
+    "@mui/icons-material": "^5.8.4",
```

Comment:
I see we have these ones added but still the old ones. Is that intended?

Reply:
Yes.  I didn't end up removing all of the old `material-ui` imports from all the component files so keeping these old dependencies is necessary for now.  I looked into changing the imports on all component files, but that complicated some stuff with the theme we are importing from shared-components.  If you want to have more discussion around this, let me know, and we can get on a call or something so I can show exactly what I was seeing.

Reply:
Looks like SSC still has the older Mui packages listed as Peer Dependencies.

Reply:
@laurelbean Correct, it does.